code/controllers/subsystem/playtime.dm

```diff
@@ -19,18 +19,26 @@ SUBSYSTEM_DEF(playtime)
 		var/mob/M = C.mob
 		var/datum/entity/player/P = C.player_data
 
-		if(!M || !M.job || M.stat == DEAD || M.statistic_exempt || !P || !P.playtime_loaded)
+		var/effective_job
+		if(!M || !P || !P.playtime_loaded || M.statistic_exempt)
+			effective_job = null
+		else if(isobserver(M))
```

Comment:
```suggestion
		else if(isobserver(M) || M.stat == DEAD)
```
This should generally just be the time spent dead rather than the time spent as an observer.

Reply:
modified to take into account non-ghosted dead people (isliving check to avoid counting new_player on lobby